src/porepy/viz/fracture_visualization.py

```diff
@@ -25,12 +25,27 @@ def plot_fractures(pts, edges, domain=None, colortag=None, **kwargs):
             (e.g. by fracture family). If provided, different colors will be
             asign to the different families. Defaults to all fractures being
             black.
+        ax (matplotlib.axes.Axes, optional): If not given an axis, an axis will be created.
```

Comment:
only one comment, why not to put ax in the kwargs? for the rest okay

Reply:
I did really have any specific reason.